task4/task4.ipynb

```diff
@@ -57,14 +74,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    ">>Should develop strategies to keep the customers in the second month"
+    "**OLD:**\n",
+    "\n",
+    "\n",
+    "*Should develop strategies to keep the customers in the second month*\n",
+    "\n",
+    "\n",
+    "*The expected lifetime of above 2 is heavily influenced by the loyal customers, that have been staying for many months and therefore driving the expected value up. On the other side we have seen big drops of customers after only one or two months, and we would therefore want to get more users past that critical point of two months. This could be done by more effectively showing the customer relevant content. It is however difficult to personalize content for a user that they have little data on, and one strategy could therefore be to gather data about the particular user from outside of Netflix itself to propose relevant, personalized content that may make the user stay past the critical point*"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The expected lifetime of above 2 is heavily influenced by the loyal customers, that have been staying for many months and therefore driving the expected value up. On the other side we have seen big drops of customers after only one or two months, and we would therefore want to get more users past that critical point of two months. This could be done by more effectively showing the customer relevant content. It is however difficult to personalize content for a user that they have little data on, and one strategy could therefore be to gather data about the particular user from outside of Netflix itself to propose relevant, personalized content that may make the user stay past the critical point"
+    "$5.29$ is a rather high expected lifetime for customers. This is using the constant retention rate and assuming everyone finishes at least one month. The math looks something like $1 + 2\\cdot 0.57^1 + 3 \\cdot 0.57^2 + 4 \\cdot 0.57^3  \\cdots = 1 + 1.14 + 0.97 + 0.74 \\cdots $. We have already discussed that the average retention rate is artificially high, and so this expected value seems high as well as a consequence. It makes however sence if we consider the long-serving customers and not only the ones acquired in our time window of data. And so this expected value rather represents one type of the general customer base which are the long-serving ones"
```

Comment:
Some flaw in this logic since we are discussing "long-serving" customers, but they have a 100% chance of surviving month 1, meaning they are newly acquired.. But with the assumption that they will be a part of the long-serving customers